static/js/chart/draw.ts

```diff
@@ -23,7 +23,9 @@ import {
 
 const NUM_X_TICKS = 5;
 const NUM_Y_TICKS = 5;
-const MARGIN = { top: 20, right: 10, bottom: 30, left: 35, yAxis: 3 };
+const MARGIN = { top: 20, right: 10, bottom: 30, left: 35, yAxis: 3, legendWidth: 200};
```

Comment:
Nit: please follow convention and leave a space before the closing bracket.

Reply:
Done